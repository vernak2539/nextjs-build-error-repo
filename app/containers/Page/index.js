import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import Head from 'next/head';

const Page = (props) => (
  <Fragment>
    <Head>
      <title>Agent Support</title>
    </Head>
    {props.children}
  </Fragment>
);

Page.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Page;
