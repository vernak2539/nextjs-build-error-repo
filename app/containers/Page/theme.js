import developmentFavicon from './images/favicons/development/favicon-32x32.png';
import stagingFavicon from './images/favicons/staging/favicon-32x32.png';
import productionFavicon from './images/favicons/production/favicon-32x32.png';

export const getFaviconAndTheme = (env) => {
  switch (env) {
    case 'production':
      return {
        favicon: productionFavicon,
        themeColor: '#009e99',
      };
    case 'staging':
      return {
        favicon: stagingFavicon,
        themeColor: '#e25d50',
      };
    default:
      return {
        favicon: developmentFavicon,
        themeColor: '#7a7a7a',
      };
  }
};
