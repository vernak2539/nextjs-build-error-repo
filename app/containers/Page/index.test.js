import React from 'react';
import { shallow } from 'enzyme';

import Page from '.';

describe('Page container', () => {
  it('renders as expected', () => {
    const container = shallow(<Page>Child</Page>);
    expect(container).toMatchSnapshot();
  });
});
