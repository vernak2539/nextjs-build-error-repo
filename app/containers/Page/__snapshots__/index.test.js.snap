// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page container renders as expected 1`] = `
<Fragment>
  <Head>
    <link
      href="asset-path"
      rel="shortcut icon"
      type="image/x-icon"
    />
    <title>
      Agent Support
    </title>
    <link
      color="#7a7a7a"
      href="asset-path"
      rel="mask-icon"
    />
    <meta
      content="#7a7a7a"
      name="theme-color"
    />
  </Head>
  Child
</Fragment>
`;
